### PR TITLE
Update primary button in demo code

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-update-primary-button-in-demo](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-update-primary-button-in-demo)

File: src/app/public/plugin-resources/code-examples/data-manager/basic/data-manager-demo.component.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <sky-data-manager>
   <sky-data-manager-toolbar>
-    <!-- <sky-data-manager-toolbar-left-item *ngIf="activeViewId === 'repeaterView'">
+    <!-- <sky-data-manager-toolbar-primary-item *ngIf="activeViewId === 'repeaterView'">
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/data-manager/basic/data-manager-demo.component.html:3]: Is this supposed to be commented out?

File: package.json [hunk 2]
```diff
@@ -40,42 +40,42 @@
...
+    "@skyux/layout": "4.6.0",
+    "@skyux/lists": "4.7.1",
+    "@skyux/lookup": "4.10.0",
+    "@skyux/modals": "4.5.2",
     "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/popovers": "4.4.0",
-    "@skyux/router": "4.0.1",
-    "@skyux/tabs": "4.5.0",
-    "@skyux/theme": "4.13.2",
+    "@skyux/popovers": "4.5.0",
```
Comment (Blackbaud-AlexKingman) [inline package.json:71]: Bump one more time to get 4.5.1

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-update-primary-button-in-demo](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-update-primary-button-in-demo)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**